src/DotNetWorker.Core/Context/Features/DefaultFunctionInputBindingFeature.cs

```diff
@@ -29,7 +29,10 @@ public async ValueTask<FunctionInputBindingResult> BindFunctionInputAsync(Functi
         {
             ObjectDisposedThrowHelper.ThrowIf(_disposed, this);
 
-            await _semaphoreSlim.WaitAsync(WaitTimeInMilliSeconds, context.CancellationToken);
+            // Setting second parameter to CancellationToken.None to prevent a TaskCancelledException if the 
+            // pipeline cancels the code before the function is invoked. This way the customer code will be invoked
+            // and they can handle the cancellation token.
+            await _semaphoreSlim.WaitAsync(WaitTimeInMilliSeconds, CancellationToken.None);
```

Comment:
Is this the only place we need to handle it? Anywhere else in the invocation pipeline the FunctioContext.CancellationToken is being used?

Reply:
Yeah I believe so. The only other place this is handled is [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/e8e4b161e608a53a7cad40c27447e2a395ac7344/src/DotNetWorker.Core/Converters/CancellationTokenConverter.cs#L15) in the `CancellationTokenConverter`, but we'd want to pass it in during this instance